Watchman.DomainModel/Warnings/Commands/AddWarnEventCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Watchman.Cqrs;
+using Watchman.DomainModel.Warnings;
+
+namespace Watchman.DomainModel.Warnings.Commands
+{
+    public class AddWarnEventCommand : ICommand
+    {
+        public WarnEvent WarnEvent { get; }
+
+        public AddWarnEventCommand(ulong receiverId, ulong granterId, string reason, ulong serverId)
+        {
+            WarnEvent = new WarnEvent(
```

Comment:
dlaczego to sie dzieje na poziomie komendy?

Reply:
jak powinno być?

Reply:
no tutaj powinieneś ustawiać propertiesy 
a nowy obiekt tworzyć na poziomie handlera 

komenda nie powinna zmieniać danych w locie a jedynie je przekazywać 1:1

Reply:
ok